tests/integration/src/test/java/org/mule/test/construct/FlowRefTestCase.java

```diff
@@ -64,62 +45,6 @@ public void dynamicFlowRef() throws Exception {
                  flowRunner("flow2").withPayload("0").withVariable("letter", "B").run().getMessageAsString(muleContext));
   }
 
-  public static class ProcessorPathAssertingProcessor implements Processor, FlowConstructAware {
-
-    private static List<String> traversedProcessorPaths = new ArrayList<>();
-    private FlowConstruct flowConstruct;
-
-    @Override
-    public Event process(Event event) throws MuleException {
-      traversedProcessorPaths
-          .add(((Flow) muleContext.getRegistry().lookupFlowConstruct(flowConstruct.getName())).getProcessorPath(this));
-      return event;
-    }
-
-    @Override
-    public void setFlowConstruct(FlowConstruct flowConstruct) {
-      this.flowConstruct = flowConstruct;
-    }
-  }
-
-  @Ignore("MULE-11482 - ignoring since it's going to be removed on next commit")
-  @Test
-  public void dynamicFlowRefProcessorPath() throws Exception {
```

Comment:
With the current approach we don't need specific test cases for this use cases since there's a common behaviour not dependant on each MP